app/views/declaration/summary/summary_page.scala.html

```diff
@@ -84,10 +85,6 @@
 
     @items_section(suppDecData.items)
 
-    @if(mode == Mode.AmendMode) {
```

Comment:
why we delete amend section ?

Reply:
It's not deleted.  Its moved to the else block

Reply:
It's moved into an `else` block